int32 id
semap_msgs/Point2DModel model
---

